terraform/projects/app-ecs-instances/main.tf

```diff
@@ -185,6 +199,16 @@ module "ecs_instance" {
   )}"
 }
 
+resource "aws_autoscaling_schedule" "asg_dev_scaledown_schedules" {
+  scheduled_action_name  = "asg_dev_scaledown_schedule-${element(var.asg_dev_scaledown_schedules, count.index)}"
```

Comment:
Just checking that you wanted to have the name contain the cron expression in it rather than just a simple `asg_dev_scaledown_schedule_0`, `asg_dev_scaledown_schedule_1` etc?

It looks a little bit strange in the AWS console but technically there's nothing wrong with it

Reply:
ok